wrappers/c/zxing.h

```diff
@@ -0,0 +1,153 @@
+#ifndef _ZXING_C_H
+#define _ZXING_C_H
```

Comment:
I was thinking about the name of the header and thought that `zxing.h` is a bit too generic for my taste in this context. Mabe `zxingc.h` or `zxing-c.h`. I can't quite decide what I prefer. I wanted to use `zxing-cpp` in the context of the python and android module but there a '-' is not allowed or problematic, so I ended up with `zxingcpp`. Do you have a convincing argument for either of them?

Reply:
Basically indifferent to me. Between `zxingc` and `zxing-c` I prefer `zxingc`, mainly because it's shorter, but also because of what you said (`zxingcpp` being used somewhere else) and in case `-` is disallowed for whatever reason.

Reply:
I checked what other libraries do. '-' is totally fine in a header file name. -> Please rename this file to `zxing-c.h` and the cpp file to `zxing-c.cpp`. This also fits nicely with your `zxing-c-test.c` choice. ;).